### PR TITLE
Modeler integration

### PR DESCRIPTION
**Pull request by Beacze**
First version of workflow page

### CODE REVIEW

File: Kitodo/src/test/java/org/kitodo/FileLoader.java [hunk 1]
```diff
@@ -216,6 +226,14 @@ public static void createRulesetFile() throws IOException {
         Files.write(Paths.get(rulesetPath), content);
     }
 
+    public static void deleteDiagramBaseFile() throws IOException {
```
Comment (henning-gerhardt) [inline Kitodo/src/test/java/org/kitodo/FileLoader.java]: Public method without JavaDoc.

File: Kitodo/src/test/java/org/kitodo/FileLoader.java [hunk 2]
```diff
@@ -216,6 +226,14 @@ public static void createRulesetFile() throws IOException {
...
+        Files.deleteIfExists(Paths.get(diagramBasePath));
+    }
+
+    public static void deleteDiagramTestFile() throws IOException {
```
Comment (henning-gerhardt) [inline Kitodo/src/test/java/org/kitodo/FileLoader.java]: Public method without JavaDoc.

File: Kitodo/src/test/java/org/kitodo/forms/ModelerFormTest.java [hunk 3]
```diff
@@ -0,0 +1,100 @@
+package org.kitodo.forms;
```
Comment (henning-gerhardt) [inline Kitodo/src/test/java/org/kitodo/forms/ModelerFormTest.java]: New test file without license header.

**Comment by Beacze** [comment]
License headers are added.

**Comment by henning-gerhardt** [comment]
Even after you added the Camunda license header to the generated files by the Camunda tool I do not think that the Camunda license is compatible with the GPLv3 license which is used in the project.

**Comment by henning-gerhardt** [comment]
Integration of this created files by the camunda tool is approved by the general management of Kitodo e.V. But the license header of this files must be adjusted and a general license file for this MIT camunda license must be added.

File: Kitodo/src/main/webapp/WEB-INF/resources/css/bpmn-js/bpmn-font/css/bpmn-codes.css [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+.bpmn-icon-connection-multi:before { content: '\e860'; } /* '' */
+.bpmn-icon-lane:before { content: '\e861'; } /* '' */
+.bpmn-icon-lasso-tool:before { content: '\e862'; } /* '' */
+.bpmn-icon-lane-insert-above:before { content: '\e863'; } /* '' */
+.bpmn-icon-lane-divide-three:before { content: '\e864'; } /* '' */
+.bpmn-icon-lane-divide-two:before { content: '\e865'; } /* '' */
+.bpmn-icon-data-input:before { content: '\e866'; } /* '' */
+.bpmn-icon-data-output:before { content: '\e867'; } /* '' */
+.bpmn-icon-hand-tool:before { content: '\e868'; } /* '' */
+.bpmn-icon-transaction:before { content: '\e8c4'; } /* '' */
```
Comment (henning-gerhardt) [inline Kitodo/src/main/webapp/WEB-INF/resources/css/bpmn-js/bpmn-font/css/bpmn-codes.css]: Missing new line at end of file.

File: Kitodo/src/main/webapp/WEB-INF/resources/css/bpmn-js/bpmn-font/css/bpmn-embedded.css [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
...Mz8ZNS8EITEjFSMPAxUfBDM3MzczFzMXMxczFzMXMx8GMz8FNS8CIy8JIycjJyMnIzUHIw8CIw8BFQcjDwEjBxUHIwcjByMHFQ8DIxUPARUPAiMPAR8FMz8dMz8DLwMFDwUfFhUfBDM/BDUnNS8INSc1LwQjJyMnNS8BIyc1JzUvATUjLwE1JyMnNS8CNS8DBSMPBRUjFQcVBxUjFQcVFxUzFRcVFxUXFR8JMz8ENS8CNSc1JzUnNSc1JzU3NTc1NzU3NTc1LwQFIw8DFQcXFQcVBxUHFQcVDwgVHwMzPwY1NzU/BzU3NTM1NzU3NSc1LwMFIw8DFRcVFxUXFRcVHwcVFxUfAjMVHwMzPwQ1Lws1JzUnNSc1JzUvBAUjDwUVBxUPDxUfBT8CNT8BNTczNzU/AjM/CjU3LwQFDwUfAhUXFRczFzMfAjMfATMVFxUXFRczHwEzFzMXFRcVHwEVHwIVHwIzPwU1LxwFIw8IIwcjByMHIwcjByMPBBUfBTM3MzczNzM3Mz8LMzczPwE1PwE1NzU/BDUvBAUPBRUfBTMfARUXMxczHwEVHwEzFxUfBDMXMx8GMz8FNS8WISMPFxUfBjM/CDM/BDU3MzczNzU/ATU/ATM/BDUvBAFdAS7+6v7+GsrKysrKysplCwYLBQwFBQYmBgoIBAIDAQECBgQJBCsFCQUFBQUKBR4FCgUFBQUKBBgJBQQDBAQBAQQDBAcmBQYFCwYLBvQFBAUDBAQCAggCDAcIBwQDBAMEAwcGBw8CCQIDEQQDBAkJBAQDBAECAwIBAgMCAQISAQkEBgEDBAMBAwgEAwEDAQMBCAQEBAEECAEECv3xBQQIBQQBBAQEDAQEBAgDAQMEAwQDBAkBDwIBAgMDBQIBBAMDAQIDAwgECQUHBQUQBgIGAg8HBgcDBAMEAwQHCAQEBwIBAgIDBAgBBBINCgUHBgICAwgIBAEIBCYECAMEBAQHBAgDGgMIBwcDCQUEBAQDBAEEAwcBAwUEBAQEBAkEJgQJBQQEBQ2tBAgGBwEDCAMBFQMBAwMBCQEMAQkJAwMCAQIDAwgEAQQCAgIDAwgEBQQJAwgDAgQDAgMCAwIDAgMDAwIDAwMDAwMDAwYDIwIDBAQCAgUECAGEBQgEAwMDAQYDAwIDAgMCAwIDBxYBBAEEAQYDAgIEBAMEBQkEBAQDBAECAgIBAgEC
```
Comment (henning-gerhardt) [inline Kitodo/src/main/webapp/WEB-INF/resources/css/bpmn-js/bpmn-font/css/bpmn-embedded.css]: Missing new line at end of file.

File: Kitodo/src/main/webapp/WEB-INF/resources/css/bpmn-js/bpmn-font/css/bpmn.css [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+.bpmn-icon-connection-multi:before { content: '\e860'; } /* '' */
+.bpmn-icon-lane:before { content: '\e861'; } /* '' */
+.bpmn-icon-lasso-tool:before { content: '\e862'; } /* '' */
+.bpmn-icon-lane-insert-above:before { content: '\e863'; } /* '' */
+.bpmn-icon-lane-divide-three:before { content: '\e864'; } /* '' */
+.bpmn-icon-lane-divide-two:before { content: '\e865'; } /* '' */
+.bpmn-icon-data-input:before { content: '\e866'; } /* '' */
+.bpmn-icon-data-output:before { content: '\e867'; } /* '' */
+.bpmn-icon-hand-tool:before { content: '\e868'; } /* '' */
+.bpmn-icon-transaction:before { content: '\e8c4'; } /* '' */
```
Comment (henning-gerhardt) [inline Kitodo/src/main/webapp/WEB-INF/resources/css/bpmn-js/bpmn-font/css/bpmn.css]: Missing new line at end of file.

File: Kitodo/src/main/webapp/WEB-INF/resources/css/bpmn-js/bpmn-font/css/bpmn-codes.css (hunk 4 above)
Comment (Beacze) [reply Kitodo/src/main/webapp/WEB-INF/resources/css/bpmn-js/bpmn-font/css/bpmn-codes.css]: Added

File: Kitodo/src/main/webapp/WEB-INF/resources/css/bpmn-js/bpmn-font/css/bpmn.css (hunk 6 above)
Comment (Beacze) [reply Kitodo/src/main/webapp/WEB-INF/resources/css/bpmn-js/bpmn-font/css/bpmn.css]: Added

**Pull request merged**